spec/cucumber_factory/steps_spec.rb

```diff
@@ -200,6 +200,15 @@
       user.deleted.should == true
     end
 
+    it "should allow to name records and set a belongs_to association to that record by refering to that name" do
```

Comment:
I saw that you call `remember_record_names` as part of the update steps. Please add either a new scenario that tries to find a record by its updated attribute or omit the call to `remember_record_names`.

e.g.
> there is a movie
> the move above has the name "my movie"
> there is a collection with the movie "my movie"